src/Browser/Json.php

```diff
@@ -110,6 +111,26 @@ public function assertThatEach(string $selector, callable $assert): void
         foreach ($value as $item) {
             $assert(self::encode($item));
         }
+
+        return $this;
+    }
+
+    public function assertMatchesSchema(string $jsonSchema): self
```

Comment:
I think we should not require a file name here, because it would prevent users to provide json "hard coded"

Reply:
Agreed